### PR TITLE
Implement top section

### PR DESCRIPTION
**Pull request by duylong13**

### CODE REVIEW

File: Practice2/src/index.html [hunk 1]
```diff
@@ -10,7 +10,9 @@
 </head>
 
 <body>
-  <section>
-  </section>
+  <div>
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/index.html]: Please remove unused div.

File: Practice2/src/partials/top-section.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<header class="page-top">
+  <div>
+    <picture>
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/partials/top-section.html]: You are using the picture tag wrong. Please check source min-width careful

File: Practice2/src/partials/top-section.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      <source media="(min-width: 1440px)" srcset="./assets/images/top-background.png">
+      <source media="(min-width: 1440px)" srcset="./assets/images/top-background-tablet.jpg">
+      <source media="(min-width: 1440px)" srcset="./assets/images/top-background-mobile.jpg">
+      <img src="./assets/images/top-background.png" alt="background" class="top-img">
+    </picture>
+  </div>
+  <div>
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/partials/top-section.html]: Remove unused div.

File: Practice2/src/partials/top-section.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<header class="page-top">
+  <div>
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/partials/top-section.html]: remove unused div

File: Practice2/src/styles/components/top.css [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+.top-img {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  z-index: -1;
+}
+
+.top-img-sub {
+  display: flex;
+  top: 312px;
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/styles/components/top.css]: Remove unused code here

File: Practice2/src/styles/components/top.css [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.play-button {
+  background-color: Transparent;
+  background-repeat: no-repeat;
+  border: none;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin: -110px -36px;
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/styles/components/top.css]: When using the absolute to center a block. You should apply margin top, margin left only

File: Practice2/src/styles/components/top.css [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.top-text {
+  text-transform: uppercase;
+  font-size: 36px;
+  line-height: var(--lh-heading-wide);
+  margin-right: 279px;
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/styles/components/top.css]: Should not use margin with a very big value like that. Let try to find another way.

File: Practice2/src/styles/components/top.css [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.nav-top {
+  display: flex;
+  padding: 48px var(--p-large) 0 var(--p-large);
+}
+
+.nav-icon {
+  padding-top: 19px;
+  margin-left: 148px;
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/styles/components/top.css]: Should not use margin with a very big value like that. Let try to find another way.

File: Practice2/src/styles/components/top.css [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.nav-icon-success {
+  margin-right: 43px;
+}
+
+.top-heading {
+  top: 163px;
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/styles/components/top.css]: Please remove unused code

File: Practice2/src/styles/components/top.css [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+  font-size: var(--fs-heading-large);
+  line-height: var(--lh-heading-large);
+  text-align: center;
+  margin-top: 74px;
+  padding: 0 410px;
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/styles/components/top.css]: Should not use padding with a very big value like that. Let try to find another way.

File: Practice2/src/styles/components/top.css [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+.nav-video {
+  position: relative;
+}
+
+@media screen and (min-width: 768px) {
+  .nav-top {
+    display: flex;
+
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/styles/components/top.css:169]: Remove line break here

**Review by vuongnguyen-agilityio — CHANGES_REQUESTED** [review]

File: Practice2/src/index.html (hunk 1 above)
Comment (duylong13) [reply Practice2/src/index.html]: Fixed

File: Practice2/src/partials/top-section.html (hunk 3 above)
Comment (duylong13) [reply Practice2/src/partials/top-section.html]: Fixed

File: Practice2/src/partials/top-section.html (hunk 4 above)
Comment (duylong13) [reply Practice2/src/partials/top-section.html]: Fixed

File: Practice2/src/styles/components/top.css (hunk 5 above)
Comment (duylong13) [reply Practice2/src/styles/components/top.css]: Everything in this is used

File: Practice2/src/styles/components/top.css (hunk 7 above)
Comment (duylong13) [reply Practice2/src/styles/components/top.css]: I'll try

File: Practice2/src/styles/components/top.css (hunk 8 above)
Comment (duylong13) [reply Practice2/src/styles/components/top.css]: Fixed

File: Practice2/src/styles/components/top.css (hunk 7 above)
Comment (duylong13) [reply Practice2/src/styles/components/top.css]: Fixed

File: Practice2/src/partials/top-section.html (hunk 2 above)
Comment (duylong13) [reply Practice2/src/partials/top-section.html]: Fixed

File: Practice2/src/partials/top-section.html [hunk 12]
```diff
@@ -0,0 +1,43 @@
+<header class="page-top">
+  <picture>
+    <source media="(min-width: 1440px)" srcset="./assets/images/top-background.png">
+    <source media="(min-width: 768px)" srcset="./assets/images/top-background-tablet.png">
+    <source media="(min-width: 320px)" srcset="./assets/images/top-background-mobile.png">
+    <img src="./assets/images/top-background-mobile.png" alt="background" class="top-img">
+  </picture>
+    <div class="nav-top">
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/partials/top-section.html:8]: Please check tabsize here

File: Practice2/src/partials/top-section.html [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+        </li>
+      </ul>
+      <div class="nav-icon">
+        <img src="./assets/images/twitter-icon.png" alt="twitter-icon" class="nav-icon-success">
+        <img src="./assets/images/facebook-icon.png" alt="facebook-icon" class="nav-icon-success">
+        <img src="./assets/images/instagram-icon.png" alt="instagram-icon">
+      </div>
+      <button class="nav-menu"><img src="./assets/images/menu-icon.png" alt="menu-icon"></button>
+    </div>
+  <div class="info-video">
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/partials/top-section.html:31]: Check tab size here

**Review by vuongnguyen-agilityio — CHANGES_REQUESTED** [review]

File: Practice2/src/partials/top-section.html (hunk 12 above)
Comment (duylong13) [reply Practice2/src/partials/top-section.html:8]: Fixed

File: Practice2/src/partials/top-section.html (hunk 13 above)
Comment (duylong13) [reply Practice2/src/partials/top-section.html:31]: Fixed

**Pull request merged**